Makefile

```diff
@@ -1,24 +1,23 @@
 .PHONY: requirements
 
 requirements: requirements.js
-	pip install -q -r requirements/base.txt --exists-action w
+	pip install .
```

Comment:
The `.` is a reference to the current directory. When `pip install` is called with a reference to a directory, it checks for a `setup.py` script inside that directory, and if it finds it, it calls `python setup.py install`, which installs the dependencies specified in the `install_requires` section of the `setup.py` file. However, if you prefer a `requirements.txt` file, I can pull that out and revert this.